.github/CODEOWNERS

```diff
@@ -15,3 +15,9 @@
 kernel/packages/build-ecs/**/*               @decentraland/kernel-team
 kernel/packages/decentraland-amd/**/*        @decentraland/kernel-team
 kernel/packages/decentraland-ecs/**/*        @decentraland/kernel-team
+
+# comms
+kernel/packages/shared/comms/logic/rfc-4-room-connection.ts   @decentraland/platform
```

Comment:
I think the reason it's showing as "contains errors" is because platform doesn't have write permissions on the repo.